### PR TITLE
Add npm provenance

### PR DESCRIPTION
**Pull request by Ethella**
### 📦 Pull Request

* Enable NPM Provenance
* Bump Lerna version

### ✅ Fixed Issues

- [List any fixed issues here like: Fixes #XXXX]

### 🚨 Test instructions

image

[Describe any additional context required to test the PR/feature/bug fix.]

### ⚠️ Don't forget to add a [semver](https://semver.org/) label! 
Please only add **one** label:
- `patch`: Bug Fix?
- `minor`: New Feature?
- `major`: Breaking Change?
- `skip-release`: It's unnecessary to publish this change.

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @magic-ext/algorand@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/aptos@5.0.1-canary.703.7755925479.0
  npm install @magic-ext/avalanche@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/bitcoin@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/conflux@15.0.1-canary.703.7755925479.0
  npm install @magic-ext/cosmos@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/ed25519@13.0.1-canary.703.7755925479.0
  npm install @magic-ext/flow@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/gdkms@5.0.1-canary.703.7755925479.0
  npm install @magic-ext/harmony@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/icon@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/near@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/oauth@16.0.1-canary.703.7755925479.0
  npm install @magic-ext/oidc@5.0.1-canary.703.7755925479.0
  npm install @magic-ext/polkadot@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/react-native-bare-oauth@19.0.1-canary.703.7755925479.0
  npm install @magic-ext/react-native-expo-oauth@19.0.1-canary.703.7755925479.0
  npm install @magic-ext/solana@19.0.1-canary.703.7755925479.0
  npm install @magic-ext/taquito@14.0.1-canary.703.7755925479.0
  npm install @magic-ext/terra@14.0.1-canary.703.7755925479.0
  npm install @magic-ext/tezos@17.0.1-canary.703.7755925479.0
  npm install @magic-ext/webauthn@16.0.1-canary.703.7755925479.0
  npm install @magic-ext/zilliqa@17.0.1-canary.703.7755925479.0
  npm install @magic-sdk/commons@18.0.1-canary.703.7755925479.0
  npm install @magic-sdk/pnp@16.0.1-canary.703.7755925479.0
  npm install @magic-sdk/provider@22.0.1-canary.703.7755925479.0
  npm install @magic-sdk/react-native-bare@23.0.1-canary.703.7755925479.0
  npm install @magic-sdk/react-native-expo@23.0.1-canary.703.7755925479.0
  npm install @magic-sdk/types@18.0.1-canary.703.7755925479.0
  npm install magic-sdk@22.0.1-canary.703.7755925479.0
  # or 
  yarn add @magic-ext/algorand@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/aptos@5.0.1-canary.703.7755925479.0
  yarn add @magic-ext/avalanche@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/bitcoin@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/conflux@15.0.1-canary.703.7755925479.0
  yarn add @magic-ext/cosmos@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/ed25519@13.0.1-canary.703.7755925479.0
  yarn add @magic-ext/flow@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/gdkms@5.0.1-canary.703.7755925479.0
  yarn add @magic-ext/harmony@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/icon@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/near@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/oauth@16.0.1-canary.703.7755925479.0
  yarn add @magic-ext/oidc@5.0.1-canary.703.7755925479.0
  yarn add @magic-ext/polkadot@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/react-native-bare-oauth@19.0.1-canary.703.7755925479.0
  yarn add @magic-ext/react-native-expo-oauth@19.0.1-canary.703.7755925479.0
  yarn add @magic-ext/solana@19.0.1-canary.703.7755925479.0
  yarn add @magic-ext/taquito@14.0.1-canary.703.7755925479.0
  yarn add @magic-ext/terra@14.0.1-canary.703.7755925479.0
  yarn add @magic-ext/tezos@17.0.1-canary.703.7755925479.0
  yarn add @magic-ext/webauthn@16.0.1-canary.703.7755925479.0
  yarn add @magic-ext/zilliqa@17.0.1-canary.703.7755925479.0
  yarn add @magic-sdk/commons@18.0.1-canary.703.7755925479.0
  yarn add @magic-sdk/pnp@16.0.1-canary.703.7755925479.0
  yarn add @magic-sdk/provider@22.0.1-canary.703.7755925479.0
  yarn add @magic-sdk/react-native-bare@23.0.1-canary.703.7755925479.0
  yarn add @magic-sdk/react-native-expo@23.0.1-canary.703.7755925479.0
  yarn add @magic-sdk/types@18.0.1-canary.703.7755925479.0
  yarn add magic-sdk@22.0.1-canary.703.7755925479.0
  ```
</details>

### CODE REVIEW

File: .github/workflows/canary.yml [hunk 1]
```diff
@@ -7,8 +7,9 @@ on:
     types: [checks_requested]
 
 env:
-  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.ARI_ADMIN_PAT_TOKEN }}
```
Comment (romin-halltari) [inline .github/workflows/canary.yml:10]: The naming here seems kinda strange, is it intentional?

**Review by patjacobs-magic — APPROVED** [review]
Looks good!

**Review by romin-halltari — APPROVED** [review]

File: .github/workflows/canary.yml (hunk 1 above)
Comment (Ethella) [reply .github/workflows/canary.yml:10]: Yes. It is a temp personal token just to have higher permission to write PR and CHANGELOG. Once infra team figure out the best way to manage these tokens, it shall get replaced

**Comment by Ariflo** [comment]
:rocket: PR was released in [`@magic-ext/algorand@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/algorand%4017.0.1), [`@magic-ext/aptos@5.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/aptos%405.0.1), [`@magic-ext/avalanche@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/avalanche%4017.0.1), [`@magic-ext/bitcoin@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/bitcoin%4017.0.1), [`@magic-ext/conflux@15.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/conflux%4015.0.1), [`@magic-ext/cosmos@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/cosmos%4017.0.1), [`@magic-ext/ed25519@13.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/ed25519%4013.0.1), [`@magic-ext/flow@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/flow%4017.0.1), [`@magic-ext/gdkms@5.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/gdkms%405.0.1), [`@magic-ext/harmony@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/harmony%4017.0.1), [`@magic-ext/icon@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/icon%4017.0.1), [`@magic-ext/near@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/near%4017.0.1), [`@magic-ext/oauth@16.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/oauth%4016.0.1), [`@magic-ext/oidc@5.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/oidc%405.0.1), [`@magic-ext/polkadot@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/polkadot%4017.0.1), [`@magic-ext/react-native-bare-oauth@19.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-bare-oauth%4019.0.1), [`@magic-ext/react-native-expo-oauth@19.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/react-native-expo-oauth%4019.0.1), [`@magic-ext/solana@19.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/solana%4019.0.1), [`@magic-ext/taquito@14.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/taquito%4014.0.1), [`@magic-ext/terra@14.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/terra%4014.0.1), [`@magic-ext/tezos@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/tezos%4017.0.1), [`@magic-ext/webauthn@16.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/webauthn%4016.0.1), [`@magic-ext/zilliqa@17.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-ext/zilliqa%4017.0.1), [`@magic-sdk/commons@18.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/commons%4018.0.1), [`@magic-sdk/pnp@16.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/pnp%4016.0.1), [`@magic-sdk/provider@22.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/provider%4022.0.1), [`@magic-sdk/react-native-bare@23.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-bare%4023.0.1), [`@magic-sdk/react-native-expo@23.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/react-native-expo%4023.0.1), [`@magic-sdk/types@18.0.1`](https://github.com/magiclabs/magic-js/releases/tag/%40magic-sdk/types%4018.0.1), [`magic-sdk@22.0.1`](https://github.com/magiclabs/magic-js/releases/tag/magic-sdk%4022.0.1) :rocket:

**Pull request merged**